cuda_core/tests/example_tests/utils.py

```diff
@@ -10,7 +10,6 @@
 import os
 import sys
 
-import cupy as cp
```

Comment:
For now I treat CuPy as an optional test dependency, so this should be removed. We're not using too much memory during tests anyway.